satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -1330,7 +1445,9 @@ datasets:
     resolution: 32000
     file_type: nc_fci_asr
     file_key: reflectance_std
-    long_name: reflectance_standard_deviation_in_segment
+    long_name: TOA Bidirectional Reflectance standard deviation
+    standard_name: toa_bidirectional_reflectance
```

Comment:
also add `units: %`, set to none in netcdf file.

Reply:
done